torch/csrc/api/include/torch/nn/functional/activation.h

```diff
@@ -139,9 +139,8 @@ inline Tensor softmax(const Tensor& input, int64_t dim,
 }
 } // namespace detail
 
-inline Tensor softmax(const Tensor& input, SoftmaxFuncOptions options,
-                      c10::optional<torch::Dtype> dtype = c10::nullopt) {
-  return detail::softmax(input, options.dim(), dtype);
+inline Tensor softmax(const Tensor& input, SoftmaxFuncOptions options) {
```

Comment:
functional.py has default values for all options in:
`def softmax(input, dim=None, _stacklevel=3, dtype=None)`
so maybe
```suggestion
inline Tensor softmax(const Tensor& input, SoftmaxFuncOptions options = {}) {
```

Reply:
replied in an earlier comment